frontend/src/components/Header/Header.tsx

```diff
@@ -74,26 +74,36 @@ function InstallationPicker(page: string) {
     const { TranslateText } = useLanguageContext()
     const [allPlantsMap, setAllPlantsMap] = useState<Map<string, string>>()
     const { installationCode, switchInstallation } = useInstallationContext()
+    const [showActivePlants, setShowActivePlants] = useState<boolean>(true)
     useEffect(() => {
-        BackendAPICaller.getEchoPlantInfo().then((response: EchoPlantInfo[]) => {
+        const plantPromise = showActivePlants ? BackendAPICaller.getActivePlants() : BackendAPICaller.getEchoPlantInfo()
+        plantPromise.then(async (response: EchoPlantInfo[]) => {
             const mapping = mapInstallationCodeToName(response)
             setAllPlantsMap(mapping)
         })
-    }, [])
+    }, [showActivePlants])
+
     const mappedOptions = allPlantsMap ? allPlantsMap : new Map<string, string>()
     return (
-        <Autocomplete
-            options={Array.from(mappedOptions.keys()).sort()}
-            label=""
-            disabled={page === 'mission'}
-            initialSelectedOptions={[installationCode.toUpperCase()]}
-            placeholder={TranslateText('Select installation')}
-            onOptionsChange={({ selectedItems }) => {
-                const mapKey = mappedOptions.get(selectedItems[0])
-                if (mapKey !== undefined) switchInstallation(mapKey)
-                else switchInstallation('')
-            }}
-        />
+        <>
+            <Autocomplete
+                options={Array.from(mappedOptions.keys()).sort()}
+                label=""
+                disabled={page === 'mission'}
+                initialSelectedOptions={[installationCode.toUpperCase()]}
+                placeholder={TranslateText('Select installation')}
+                onOptionsChange={({ selectedItems }) => {
+                    const mapKey = mappedOptions.get(selectedItems[0])
+                    if (mapKey !== undefined) switchInstallation(mapKey)
+                    else switchInstallation('')
+                }}
+            />
+            <Checkbox
```

Comment:
The flotilla logo, autocomplete and checkbox are not aligned to the middle, can you just wrap them in something with an align item to the center? Seems to work good otherwise, it's just that